### PR TITLE
{lang}[foss/2020b,fosscuda/2020b] R v4.0.5

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/R/R-4.0.5-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,2852 @@
...
+    ('Rvmmin', '2018-4.17', {
+        'checksums': ['d53ba7ab06596a47990caf101a50935b2b34402f9dd8414f098a873026ff1f56'],
+    }),
+    ('Rcgmin', '2013-2.21', {
+        'checksums': ['a824a09c32d7565a3e30607c71333506d5b7197478fbe8b43f8a77dad6c12f0a'],
+    }),
+    ('optimr', '2019-12.16', {
+        'checksums': ['73b1ed560ffd74599517e8baa4c5b293aa062e9c8d50219a3a24b63e72fa7c00'],
+    }),
+    ('DMCfun', '1.3.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-4.0.5-foss-2020b.eb:2847]: @robert-mijakovic Please also include the patch that was added in #12583 (same in both easyconfigs)

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/R/R-4.0.5-foss-2020b.eb (hunk 1 above)
Comment (robert-mijakovic) [reply easybuild/easyconfigs/r/R/R-4.0.5-foss-2020b.eb:2847]: Done!

**Comment by verdurin** [comment]
`Checksum verification for extension source mvabund_4.1.9.tar.gz failed`

**Comment by boegel** [comment]
> `Checksum verification for extension source mvabund_4.1.9.tar.gz failed`

That's fixed with the change in #13020

@robert-mijakovic Can you add the alternate checksum for `mvabund` to both these easyconfigs?

**Comment by robert-mijakovic** [comment]
> > `Checksum verification for extension source mvabund_4.1.9.tar.gz failed`
> 
> That's fixed with the change in #13020
> 
> @robert-mijakovic Can you add the alternate checksum for `mvabund` to both these easyconfigs?

Done with the latest commit.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2617.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/4080ffcd42d4aeb067437475fda745a5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3300.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/eb1943a8fd8b8d5a4c5c5d8db0b71e6f for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**